modlauncher/src/main/java/net/neoforged/accesstransformer/ml/AccessTransformerService.java

```diff
@@ -0,0 +1,30 @@
+package net.neoforged.accesstransformer.ml;
+
+import cpw.mods.modlauncher.serviceapi.ILaunchPluginService;
+import net.neoforged.accesstransformer.api.AccessTransformerEngine;
+import org.objectweb.asm.Type;
+import org.objectweb.asm.tree.ClassNode;
+
+import java.util.EnumSet;
+
+public class AccessTransformerService implements ILaunchPluginService {
+    public final AccessTransformerEngine engine = AccessTransformerEngine.newEngine();
```

Comment:
How is one meant to add ATs to this engine? Or did I miss something?

Reply:
See https://github.com/neoforged/AccessTransformers/pull/4/files#diff-2a81bc6215c17e244417d7fb17bb7f7e96e2a0caf15a8c27415abeb0049ef5c1R64.

Reply:
Yeah, missed the `public` keyword